src/dense.jl

```diff
@@ -32,18 +35,23 @@ end
 
 """
 
-A solver specialized in dense output.  It wraps around a
-`Problem` and stores the subsequent steps generated by `Problem` and
-interpolates the results on request (currently this means at the
-output times stored in `opts.tout`).
+A solver specialized in dense output; it wraps an integrator. It
+stores the subsequent steps generated by `Problem` and interpolates
+the results (currently this means at the output times stored in
+`opts.tout`).
 
 """
 immutable DenseOutput{I<:AbstractIntegrator,OP<:DenseOptions} <: AbstractSolver
     integ::I  # TODO: Maybe this should be relaxed to a AbstractSolver?
               #       Then we could have a DenseOutput{DenseOutput{RK}}, say!
+              #  pwl: Or DenseOutput{StiffnessSwitching{whatever}}
     opts::OP
 end
 
+# TODO: this is confusing, firs you call `solve` with `DenseOutput{I}`
+# and then you call construct it as `DenseOutput{T}`.  Also this goes
+# against the convention that we pass as much as possible as
+# options.  What if a Solver takes more than one parameter?
 function solve{I}(ivp::IVP,
                   ::Type{DenseOutput{I}};
```

Comment:
when did that happen?  We were using `method=` keyword for selecting the integrator.